packages/backend/server/src/core/auth/next-auth.controller.ts

```diff
@@ -58,7 +58,7 @@ export class NextAuthController {
   @Throttle({
     default: {
       limit: 60,
-      ttl: 60,
+      ttl: 60 * 1000,
```

Comment:
apis has changed a lot since last time i config the throttler, since you have mention it, i plan to reorganize the api throttle configuration except auth in this pr

Reply:
previously due to frequent of session request and swr's re-validation, i have significantly relax the throttlor limit, its time to adjust